### DISCUSSION TITLE
Problem with adonisjs + mysql + docker

### DISCUSSION THREAD

**Discussion by dvadvaa** [adonisjs/core#2596] [Help] [answered]
Hello. I needed to run a project in Docker. No matter how I change .env to change host for database, it always leads to 127.0.0.1:3306

My docker-compose: https://pastebin.com/raw/XcE4CfFy
My .env: https://pastebin.com/raw/D7aFx8P3

Docker container log:
```
[1620832890219] INFO (mmr_new/1 on d0c2cb5b7cfe): started server on 0.0.0.0:3333
[1620832916297] ERROR (mmr_new/1 on d0c2cb5b7cfe): connect ECONNREFUSED 127.0.0.1:3306
```

I do not know what to do, where does it get the address 127.0.0.1, if I have a completely different one in the config?

**Comment by mastermunj** [comment, marked as answer]
Is it possible for you to create a repo replicating this problem?

I have setup similar to yours with postgres and it works properly.

**Reply by dvadvaa** [reply]
The problem was in the .env file

It was:
```
DB_HOST=adonis-mysql
DB_PORT=3306
DB_USER=root
DB_PASSWORD=password
DB_DATABASE=adonis-mysql
```
Become:
```
MYSQL_HOST=adonis-mysql
MYSQL_PORT=3306
MYSQL_USER=root
MYSQL_PASSWORD=bM3kR8kX
MYSQL_DB_NAME=adonis-mysql
```
